### ISSUE TITLE
Push OMI SO2 updated imagery

### ISSUE THREAD

**Issue by jejeking** [nasa-gibs/worldview#492]
GIBS is going to push updated colormaps for updated OMI SO2 imagery. WV is going to need to rebuild against those updated colormaps. Since we need to do the switch from provider to GIBS to WV at the same time, I need to plan with the WV team once they know when they plan on building and deploying a new WV build.

I still need to set-up GIBS UAT and get feedback from the provider but when this last hurdle is cleared, we will be ready to push to OPS.
Please let me know when you plan to schedule the next build/deployment for WV.

Thanks!

**Comment by Benjaki2** [comment]
@jejeking We plan to make a minor release early next week and likely another the following week.

Are the updated colormaps available for us to play with on SIT?

**Comment by jejeking** [comment]
https://sit.gibs.earthdata.nasa.gov/colormaps/v1.3/ as long as my colleague don't overwrite them ;-)

**Comment by localjo** [comment]
@jejeking Has GIBS pushed these updated colormaps already?

**Comment by jejeking** [comment]
Not yet. Maybe next week. I'll update this ticket once we are ready.

**Comment by minniewong** [comment]
hi @jejeking - any news on this?

**Comment by jejeking** [comment]
We are waiting on feedback from the GESDISC group for the OMI STD. Once we move forward with the STD we will be able to complete OMI NRT.

**Comment by minniewong** [comment]
Waiting for GIBS to resolve Giovanni processing errors with GESDISC

**Comment by mcechini** [comment]
I vote that we close this ticket.  It's something that does need to be addressed at some point, but is not a high priority for now, nor have any users complained.  Whenever we get around to fixing this, we can re-open a new ticket.  WV will just need to rebuild to pick up the new colormap.
